File: src/client/kotlin/dev/hybridlabs/aquatic/client/render/block/entity/AnemoneBlockEntityRenderer.java
package dev.hybridlabs.aquatic.client.render.block.entity;

import dev.hybridlabs.aquatic.block.entity.AnemoneBlockEntity;
import dev.hybridlabs.aquatic.client.model.block.entity.AnemoneBlockEntityModel;
import net.minecraft.client.render.block.entity.BlockEntityRendererFactory;
import software.bernie.geckolib.renderer.GeoBlockRenderer;

public class AnemoneBlockEntityRenderer extends GeoBlockRenderer<AnemoneBlockEntity> {
    public AnemoneBlockEntityRenderer(BlockEntityRendererFactory.Context context) {
        super(new AnemoneBlockEntityModel());
    }
}

File: src/client/kotlin/dev/hybridlabs/aquatic/client/render/block/entity/AnemoneBlockEntityRenderer.kt
package dev.hybridlabs.aquatic.client.render.block.entity;

import dev.hybridlabs.aquatic.block.entity.AnemoneBlockEntity;
import dev.hybridlabs.aquatic.client.model.block.entity.AnemoneBlockEntityModel;
import net.minecraft.client.render.block.entity.BlockEntityRendererFactory.Context;
import software.bernie.geckolib.renderer.GeoBlockRenderer;

class AnemoneBlockEntityRenderer(context: Context) : GeoBlockRenderer<AnemoneBlockEntity>(AnemoneBlockEntityModel()) {

}
